lib/finitomata.ex

```diff
@@ -1322,6 +1459,43 @@ defmodule Finitomata do
         end
       end
 
+      @spec fork(Transition.state(), State.t()) ::
+              {:noreply, State.t()}
+              | {:noreply, State.t(), :hibernate}
+      defp fork(fork_state, state) do
+        @__config__.forks
+        |> Keyword.fetch!(fork_state)
+        |> List.wrap()
+        |> safe_on_fork(fork_state, state)
+        |> tap(fn
```

Comment:
why tap if the result of this is ignored?

Reply:
|> case do
end

Reply:
Yeah, it’s a leftover, thanks!

Reply:
https://github.com/am-kantox/finitomata/issues/118